### PR TITLE
readme: update structure and add refs

### PR DESCRIPTION
**Pull request by umarcor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +19,28 @@ Proposal to define an XML-based logging format for outputs from EDA tools and lo
...
 * Coverage Model
   * Dimension
   * Bin
     * Item
     * Range
+
+### Synthesis
+
+* Area/resources
+* Timing
```
Comment (Paebbels) [inline README.md:33]: * output messages in structured format
  * categorize: message IDs
  * categorize: errors, warnings, infos

File: README.md [hunk 2]
```diff
@@ -22,10 +19,28 @@ Proposal to define an XML-based logging format for outputs from EDA tools and lo
...
+
+### Implementation
+
+* Area/resources
+* Timing
+
+# References
+
```
Comment (Paebbels) [inline README.md:41]: structured output format from Xilinx Vivado

File: README.md [hunk 3]
```diff
@@ -2,18 +2,15 @@
 
 Proposal to define an XML-based logging format for outputs from EDA tools and logging libraries.
 
-**Report Kinds**  
-* [Test Report Summary](#test-report-summary)
```
Comment (Paebbels) [inline README.md:6]: Don't we need a ToC or does GH Markdown support a ToC instruction like GitLab?

Comment (umarcor) [reply README.md:6]: As commented in a meeting, GitHub provides a ToC for any markdown source.: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/.

**Review by umarcor — COMMENTED** [review]
I updated the content. Now, a brief summary explains the input formats that are expected to be used, and there is a list of tools.

File: README.md (hunk 1 above)
Comment (umarcor) [reply README.md]: Now explained in the summary.

File: README.md (hunk 2 above)
Comment (umarcor) [reply README.md:73]: The summary now explains that multiple log/report formats are expected to be readed. Let's not specify which formats are used for Vivado, until we can double check the features in the latest versions.

**Pull request merged**